liblangutil/Exceptions.cpp

```diff
@@ -71,3 +71,14 @@ Error::Error(
 	if (!_description.empty())
 		*this << util::errinfo_comment(_description);
 }
+
+static std::optional<Severity> severityFromString(std::string _severity)
```

Comment:
This file has `using namespace std` on top so the `std::` prefix is superfluous. It's only needed in header files.
```suggestion
static optional<Severity> severityFromString(string _severity)
```

Reply:
👍